src/components/v2v3/V2V3Project/V2V3ProjectSettings/pages/ProjectUpgradesPage/UpgradeWizard/steps/FundingCycleStep/hooks/LaunchFundingCyclesData.ts

```diff
@@ -80,10 +80,17 @@ export function useLaunchFundingCyclesData({
     return
   }
 
+  // set new v3 funding cycle start to be:
+  //     `current V2 FC start time + `current V2 FC duration`
+  const newStart = fundingCycle.start.add(fundingCycle.duration)
```

Comment:
@johnnyd-eth this is good, can you add some note in the UI that explains this?

Something like this:

```
Your V2 project's next funding cycle is scheduled to start on x. Your V3 FC will start at the same time by default.
```

Approving in advance

Reply:
Good call. What do you think about just altering the current `startTime` field like I've done here? 

<img width="792" alt="Screen Shot 2022-12-12 at 11 59 48 am" src="https://user-images.githubusercontent.com/96150256/206944979-9c2a238d-2ea6-49c0-bece-3a0b00d2eb42.png">

Reply:
Done this (per Discord convo): 

https://user-images.githubusercontent.com/96150256/206952675-8d45f498-6672-4bcc-ad67-1d1afe352fa2.mp4